ci/gpu/build.sh

```diff
@@ -118,3 +118,8 @@ else
         py.test --cache-clear -vs `python -c "import distributed.diagnostics.tests.test_nvml as m;print(m.__file__)"`
     fi
 fi
+
+if [ -n "\${CODECOV_TOKEN}" ]; then
```

Comment:
I guess you have to _un-escape_ `$` here as well. The condition is always `True` otherwise.
```suggestion
if [ -n "${CODECOV_TOKEN}" ]; then
```